packages/react-components/react-combobox/docs/Migration.md

```diff
@@ -0,0 +1,176 @@
+# @fluentui/react-combobox Migration Guide
+
+## Migration from v8 Dropdown
+
+The v8 Dropdown component should be replaced with the [v9 Select](https://github.com/microsoft/fluentui/blob/master/packages/react-select/Spec.md) or v9 Dropdown component. The biggest change is that the v8 `options: IDropdownOption[]` prop should be mapped to children in v9.
```

Comment:
```suggestion
The v8 Dropdown component should be replaced with the [v9 Select](https://github.com/microsoft/fluentui/blob/master/packages/react-select/Spec.md) or v9 Dropdown component. The biggest change is that the v8 `options: IDropdownOption[]` prop is mapped to children in v9.
```

Reply:
You have a link to v9 Select. Is there a link to v9 Dropdown?

Reply:
Ah, I didn't add links to the controls in this package (so Combobox and Dropdown), since I assumed they were already at this package's docs, and I don't have separate specs for Combobox and Dropdown.